other-docs/guides/upgrading/v22.md

```diff
@@ -43,20 +43,20 @@ Refer to our [PHP Version Guide](docs://guides/updating-php/) for up-to-date com
 
 ## Headline Features
 
-### WordPress 6.7.1
+### WordPress 6.7.2
 
-WordPress 6.7.1 is a fast-follow maintenance release that addresses 16 bugs introduced in WordPress 6.7. This update focuses on improving stability across Core and the Block Editor.
+WordPress 6.7.2 is a maintenance release that addresses 35 bugs introduced in WordPress 6.7, improving stability across Core and the Block Editor. This release also includes a fix for the Twenty Twenty-Five theme, leading to an updated version of the theme being released.
```

Comment:
We don't actually ship the theme, so it doesn't need mentioning.

Reply:
Does this even need mentioning? We already had 6.7 in the last release, I don't think we need to mention minor updates here.

Reply:
Ah, okay. Removed!

Reply:
@rmccue Hm, okay, I can remove it but we did mention it in the beta announcement and upgrade guide? would leaving it out of the final release guide not seem inconsistent at this point?

Reply:
I do think we shouldn't have added the post, but doesn't matter now; let's remove here